.github/workflows/check.yml

```diff
@@ -80,6 +83,11 @@ jobs:
       # Requires `make build` so that `actions-report-image-sizes` is built.
       - name: Run the GitHub Actions linter
         run: make lint-actions
+      # Requires Git history and yarn install
+      - name: Check if a changeset is required
```

Comment:
It would be good to have this as a local development option, like `make lint-changeset`.

Reply:
The issue is that this depends on what is the base ref. Note that you can easily do this locally with `yarn changeset status`, while CI requires the `--since` argument to have a proper base.

Reply:
> It would be good to have this as a local development option, like make lint-changeset.

Do you think you'd actually run this? Personally: I never would.

Reply:
I added to Makefile so `make ci` can include this. But I agree with @nwalters512, I don't expect this to be used in isolation.